controllers/onload_controller_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	. "github.com/onsi/gomega"
 	. "github.com/onsi/gomega/gstruct"
 	"go.uber.org/mock/gomock"
+	"golang.org/x/exp/slices"
```

Comment:
```suggestion
	"slices"
```
Since we are using go >= 1.21 the `slices` package is now in the standard library.

Reply:
Oh, thanks. I didn't notice it was added like that.